mw-webapp/src/dataAccessLogic/PreviewToDTOConverter/WayPreviewToWayDTOConverter.ts

```diff
@@ -1,65 +1,24 @@
-import {Timestamp} from "firebase/firestore";
 import {WayPreview} from "src/model/businessModelPreview/WayPreview";
 import {WayDTO, WayDTOSchema} from "src/model/DTOModel/WayDTO";
 
-/**
- * WayDTO props
- */
-interface WayDTOProps {
-
-  /**
-   * @User.uuid
-   */
-  ownerUuid: string;
-
-  /**
-   * @Goal.uuid
-   */
-  goalUuid: string;
-
-  /**
-   * @User.uuids
-   */
-  mentorsUuids: string[];
-
-  /**
-   * @User.uuids
-   */
-  mentorRequestUuids: string[];
-
-  /**
-   * Last day when way was updated in ms
-   */
-  lastUpdate: Timestamp;
-
-  /**
-   * User's uuids @user.uuid for whom this way are favorite
-   */
-  favoriteForUserUuids: string[];
-
-  /**
-   * Date when way was created in ms
-   */
-  createdAt: Timestamp;
-}
+export type WayDTOProps = Pick<WayDTO,
+"ownerUuid" | "goalUuid" | "mentorRequestUuids" | "mentorUuids" | "lastUpdate" | "createdAt"
+>
 
 /**
  * Convert {@link wayPreview} to {@link WayDTO}
  */
 export const wayPreviewToWayDTOConverter = (wayPreview: WayPreview, wayDTOProps: WayDTOProps): WayDTO => {
-  const validatedWayDTO = WayDTOSchema.parse({
+  const wayDTO: WayDTO = {
     uuid: wayPreview.uuid,
     name: wayPreview.name,
     dayReportUuids: wayPreview.dayReportUuids,
-    ownerUuid: wayDTOProps.ownerUuid,
-    goalUuid: wayDTOProps.goalUuid,
-    mentorUuids: wayDTOProps.mentorsUuids,
-    mentorRequestUuids: wayDTOProps.mentorRequestUuids,
     isCompleted: wayPreview.isCompleted,
-    lastUpdate: wayDTOProps.lastUpdate,
-    favoriteForUserUuids: wayDTOProps.favoriteForUserUuids,
-    createdAt: wayDTOProps.createdAt,
-  });
+    favoriteForUserUuids: wayPreview.favoriteForUserUuids,
+    wayTags: wayPreview.wayTags,
+    jobTags: wayPreview.jobTags,
+    ...wayDTOProps,
+  };
 
-  return validatedWayDTO;
+  return WayDTOSchema.parse(wayDTO);
 };
```

Comment:
Please, add empty line at the end of file

Reply:
> Please, add empty line at the end of file

Can you please configure eslint so it will did it?

Reply:
Done.